charts/acp/templates/configmap.yaml

```diff
@@ -39,6 +39,12 @@ data:
       {{- else }}
       url: "postgres://root@{{ .Release.Name }}-cockroachdb-public:26257/defaultdb?sslmode=disable"
       {{- end }}
+    timescaledb-single:
+      {{- with .Values.timescaledb }}
+        {{- toYaml . | nindent 6 }}
+      {{- else }}
+      url: "postgres://postgres:PaSsW0rD@{{ .Release.Name }}.{{ .Release.Namespace }}.svc.cluster.local/postgres?sslmode=require"
```

Comment:
@holowinski do we want to move this to the secretconfig.yaml? Or just add proper information